projects/ghostscript/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://ghostscript.com"
+primary_contact: "skau@google.com"
+auto_ccs:
+  - "skau@google.com"
+sanitizers:
+  - address
```

Comment:
What about `undefined` and `memory`? Have you experienced any issues with them? You can just try enabling them in a separate PR and we'll see whether Travis CI recognizes any issues or they look good.

Reply:
We can follow up with memory.  I've noticed that undefined is pretty noisy for Ghostscript so I'm inclined to avoid turning it on until we can convince the upstream to fix some of their problems.